package.json

```diff
@@ -22,8 +22,7 @@
   },
   "dependencies": {
     "hubot": "^2.19.0",
-    "hubot-discord": "https://github.com/idehub/hubot-discord",
-    "hubot-discord-ng": "^1.2.35",
```

Comment:
I didn't intentionally remove the `hubot-discord-ng` dependency. I assume it was remove when I re-installed `hubot-discord`.